lib/rules/no-get-with-default.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const types = require('../utils/types');
+
+const ERROR_MESSAGE = 'Use `\|\|` or the ternary operator instead of `getWithDefault()`'; // eslint-disable-line no-useless-escape, prettier/prettier
```

Comment:
We shouldn't escape the pipes here. This error message is not part of the README.